docs/src/components/TabsPropControls.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import { TabsProps } from '@aws-amplify/ui-react';
+import { FieldLabeler } from './FieldLabeler';
+import { DemoBox } from './DemoBox';
+
+export interface TabsPropControlsProps extends TabsProps {
+  setDefaultTab: (value: React.SetStateAction<TabsProps['defaultTab']>) => void;
+  setGrow: (value: React.SetStateAction<TabsProps['grow']>) => void;
+  setJustifyContent: (
+    value: React.SetStateAction<TabsProps['justifyContent']>
+  ) => void;
+}
+
+interface TabsPropControlsInterface {
+  (props: TabsPropControlsProps): JSX.Element;
+}
+
+export const TabsPropControls: TabsPropControlsInterface = ({
+  defaultTab,
+  setDefaultTab,
+  grow,
+  setGrow,
+  justifyContent,
+  setJustifyContent,
+}) => {
+  return (
+    <DemoBox primitiveName="Tabs">
+      <FieldLabeler id="defaultTab">
+        <select
+          name="defaultTab"
+          id="defaultTab"
+          value={defaultTab}
+          onChange={(event) =>
```

Comment:
For some reason, the `defaultTab` prop is not getting applied in the demo, but works in the Usage section of the docs.

Reply:
I think that's because once the tabs have rendered, that prop is no longer acted upon.

Reply:
Updated the demo to use `currentIndex` and `onChange` so the demo controls work!